scripts/biomedical/STITCH/tmcf/drugs.tmcf

```diff
@@ -0,0 +1,21 @@
+Node: E:drug_info->E1
+typeOf: dcs:Drug
+dcid: C:drug_info->pubchem_dcid
+name: C:drug_info->pubchem_ID
+commonName: C:drug_info->name
+flatChemicalID: C:drug_info->flat_chemical_id
+stereoChemicalID: C:drug_info->stereo_chemical_id
+sourceChemicalID: C:drug_info->source_cid
+inChIKey: C:drug_info->inchikey
+atc: C:drug_info->ATC_code
+bindingDBID: C:drug_info->BindingDB_code
+chebiID: C:drug_info->ChEBI_code
+drugBankID: C:drug_info->DrugBank
+keggCompoundID: C:drug_info->KEGG_code
+pubChemSimilarSubstanceID: C:drug_info->PS_code
```

Comment:
@pradh The similarPubChemSubstanceID and similarPubChemCompoundID are both properties in which the IDs associated with them are for similar compounds but aren't exact matches. We denote this by adding similar to the front of the property - is this the right way to indicate this?